.env.template

```diff
@@ -1,2 +1 @@
-NODE_PATH=
-REACT_APP_CHAIN_ID=
\ No newline at end of file
+REACT_APP_CHAIN_ID=

```

Comment:
I dont think we need to change this file in this PR. I am also less inclined to remove NODE_PATH because it is used in the docker containers